frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -147,17 +147,14 @@
                                                 vm.loaderTitle = '';
                                                 vm.loaderContainer = angular.element('.exist-team-card');
 
-                                                // show loader
-                                                vm.startLoader("Loading Teams");
-                                                // loader end
 
                                                 parameters.url = 'challenges/challenge/' + vm.challengeId + '/participant_team/' + vm.teamId;
                                                 parameters.method = 'POST';
                                                 parameters.callback = {
                                                     onSuccess: function() {
                                                         vm.isParticipated = true;
                                                         $state.go('web.challenge-main.challenge-page.submission');
-                                                        vm.stopLoader();
+                                                        vm.isExistLoader = false;
                                                     },
```

Comment:
Same as above for hiding the loader we just set the `ng-show` variable to false.